test/lib/ansible_test/_data/requirements/sanity.pylint.txt

```diff
@@ -1,11 +1,10 @@
 # edit "sanity.pylint.in" and generate with: hacking/update-sanity-requirements.py --test pylint
-astroid==3.0.0
-dill==0.3.7
-isort==5.12.0
+astroid==3.1.0
+dill==0.3.8
+isort==5.13.2
 mccabe==0.7.0
-platformdirs==3.11.0
-pylint==3.0.1
+platformdirs==4.2.0
+pylint==3.1.0
 PyYAML==6.0.1
 tomli==2.0.1
-tomlkit==0.12.1
-typing_extensions==4.8.0
```

Comment:
It looks like this was updated on a newer Python than 3.10, since it's missing `typing_extensions`. That requirement should be updated, not removed.